app/ide-desktop/lib/dashboard/src/authentication/src/authentication/cognito.ts

```diff
@@ -27,6 +27,11 @@ const MESSAGES = {
         userNotConfirmed: 'User is not confirmed. Please check your email for a confirmation link.',
         incorrectUsernameOrPassword: 'Incorrect username or password.',
     },
+    forgotPassword: {
+        userNotFound: 'User not found. Please register first.',
```

Comment:
User is rather found, as he is sitting in front of the computer ;) This message should probably refer "User name". I know this is a minor thing and it's very picky, but this is user-facing messaging and it's important to be very correct there :)